browser/infobars/sources.gni

```diff
@@ -21,6 +21,9 @@ if (!is_android) {
     "//brave/browser/infobars/request_otr_infobar_delegate.h",
     "//brave/browser/infobars/sync_cannot_run_infobar_delegate.cc",
     "//brave/browser/infobars/sync_cannot_run_infobar_delegate.h",
+
+    # "//brave/browser/infobars/brave_psst_infobar_delegate.cc",
+    # "//brave/browser/infobars/brave_psst_infobar_delegate.h",
```

Comment:
Leftover comments

Reply:
@vadimstruts in case you're still working on this PR and splitting it out, can we mark the PR as draft for now?